src/lib/storage/BaseRecord.ts

```diff
@@ -0,0 +1,37 @@
+import uuid from 'uuid/v4';
+
+export enum RecordTypes {
```

Comment:
In TS is naming convention to name enums as singular see https://www.typescriptlang.org/docs/handbook/enums.html. Also, consider using `PascalCase`